Automattic/jetpack
# Launchpad: Added 'Connect your social media accounts' launchpad task to newsletter lists

Closes https://github.com/Automattic/wp-calypso/issues/79791

## Proposed changes:

* This PR adds a new task to Launchpad. It;s a simple task that is completed when `/marketing/connections/:siteSlug` is visited.
* It doesn't require a Calypso PR because it can take advantage of the `calypso_path` feature.

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Create a new site with `newsletter` intent
* Check that there is a new `Connect your social media accounts` task
* Clicking the task should take you to `/marketing/connections/:siteSlug`
* The task should now be marked as completed.
